components/react/Header.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-ignore
```

Comment:
Why are we ts-ignore'ing all the react files?

Reply:
Oh, that line was there before and I forgot to check why it was there.

I don't think we should delete the React components, even Shoelace have React support. Now that we have everything working you just need to add a wrapper with `createComponent` and an export in `components/react/index.js`.